### DISCUSSION TITLE
Question on the test steps - Video Capture folder

### DISCUSSION THREAD

**Discussion by tpawyant** [MerlinCooper/iRacingReplayDirector#113] [Q&A]
Hello, I am a little confused on what folder this step is asking for?  

Browse to the directory your Video Capture software uses when it captures a video file. You may have to cut/paste the directory name here.

Can someone help what should be in the path?  thank you

**Comment by tpawyant** [comment]
this is the error I am getting



![error](https://github.com/MerlinCooper/iRacingReplayDirector/assets/119341448/2780a5ab-e510-43e8-a8df-5b0c1ef50f34)

**Reply by MerlinCooper** [reply]
Most likely the verification failes because your recording software isn't configured to start recording when "Alt+F9" is sent. 

The folder should be the one where the recording software stores the recorded files.

**Comment by CzarKilo** [comment]
I'm not having the same issue however it has to do with the Video Capture step when setting the program up. The attached screenshot is the error I'm getting. I unblocked the .dll files as mentioned in [this issue.](https://github.com/MerlinCooper/iRacingReplayDirector/issues/91#issuecomment-1417442193) but continue to get the same error.
![image](https://github.com/user-attachments/assets/15e2d369-7528-43f7-80ba-c8c174495a96)
